cortx-monitorQuickstartGuide.md

```diff
@@ -28,7 +28,7 @@ This guide provides a step-by-step walkthrough for getting you CORTX-Monitor Ser
     
      1. Run the command:
             
-            `$ curl https://raw.githubusercontent.com/Seagate/cortx-prvsnr/dev/cli/src/cortx-prereqs.sh?token=APA75GY34Y2F5DJSOKDCZAK7ITSZC -o cortx-prereqs.sh; chmod a+x cortx-prereqs.sh`
+            `$ curl https://raw.githubusercontent.com/Seagate/cortx-re/main/scripts/third-party-rpm/install-cortx-prereq.sh?token=APBA7EBWCTPTEOH5PPWFYYTBCN23Y --o cortx-prereqs.sh; chmod a+x cortx-prereqs.sh`
```

Comment:
`curl http://cortx-storage.colo.seagate.com/releases/cortx/third-party-deps/rpm/install-cortx-prereq.sh --o cortx-prereqs.sh; chmod a+x cortx-prereqs.sh`
Link could be more readable.

Reply:
without token value, curl is giving 404 error.

Reply:
Okay.